app-frontend/src/app/components/projectItem/projectItem.html

```diff
@@ -1,66 +1,61 @@
-<div class="list-group-item with-rows">
-  <div><h4 class="color-dark">{{$ctrl.project.name}}</h4></div>
-  <div class="list-group-item-content">
-    <div class="list-group-overflow project-preview-container" ng-if="!$ctrl.project.scenes">
-      This project doesn't have any scenes yet
-    </div>
-    <div class="list-group-overflow project-preview-container" ng-if="$ctrl.project.scenes">
-      <rf-static-map map-id="{{$ctrl.project.id}}-map"></rf-static-map>
-    </div>
-    <div class="list-group-right">
-      <div class="font-size-small">
-        <div>
-          <div class="dropdown btn-group" uib-dropdown ng-if="!$ctrl.isSelectable">
-            <a class="btn btn-default" ui-sref="projects.edit({projectid: $ctrl.project.id})">
-              Edit
+<div class="panel-body">
+  <h4 class="project-title">{{$ctrl.project.name}}</h4>
+  <div class="row">
+    <div class="column noflex">
+      <div class="dropdown btn-group" uib-dropdown ng-if="!$ctrl.isSelectable">
```

Comment:
With the gray background of the project item, the white-text on gray button style looks a bit like a disabled style.